### ISSUE TITLE
Style/EmptyMethod with expanded style autocorrects cause warning

### ISSUE THREAD

**Issue by ofk** [bbatsov/rubocop#4420]
## Expected behavior

The `Style/EmptyMethod` ( `EnforcedStyle: expanded` ) cop corrects without warnings.

## Actual behavior

It corrects, but it causes `Lint/DefEndAlignment` warning.

## Steps to reproduce the problem

```yaml
# .rubocop.yml
Style/EmptyMethod:
  EnforcedStyle: expanded
```

```rb
# test.rb
class T
  def m; end
end
```

```
$ rubocop -Da
Inspecting 2 files
.W

Offenses:

test.rb:3:3: C: [Corrected] Style/EmptyMethod: Put the end of empty method definitions on the next line.
  def m; end
  ^^^^^^^^^^
test.rb:4:1: W: Lint/DefEndAlignment: end at 4, 0 is not aligned with def at 3, 2.
end
^^^

2 files inspected, 2 offenses detected, 1 offense corrected
```

## RuboCop version

```
$ rubocop -V
0.49.0 (using Parser 2.4.0.0, running on ruby 2.4.0 x86_64-darwin16)
```
